### PR TITLE
Release v5.5.0 rc3

### PR DESCRIPTION
**Pull request by nuttycom**
[diff from v5.5.0-rc2](https://github.com/zcash/zcash/compare/v5.5.0-rc2..195ba69) for ease of review.

### CODE REVIEW

File: doc/release-notes/release-notes-5.5.0-rc3.md [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+
+RPC Changes
+-----------
+
+- `getdeprecationinfo` has several changes:
+  - It now returns additional information about currently deprecated and
+    disabled features.
+  - A new `end_of_service` object that contains both the block height for
+    end-of-service and the estimated time that the end-of-service halt is
+    expected to occur. Note that this height is just an approximation and
```
Comment (r3ld3v) [inline doc/release-notes/release-notes-5.5.0-rc3.md:12]: I think it should be “Note that this time…”

**Review by r3ld3v — COMMENTED** [review]
Slight typo in the Release Notes.

File: doc/release-notes/release-notes-5.5.0-rc3.md (hunk 1 above)
Comment (daira) [reply doc/release-notes/release-notes-5.5.0-rc3.md:12]:
```suggestion
    expected to occur. Note that this time is just an approximation and
```

**Review by daira — APPROVED** [review]
utACK

File: doc/release-notes/release-notes-5.5.0-rc3.md (hunk 1 above)
Comment (nuttycom) [reply doc/release-notes/release-notes-5.5.0-rc3.md]: @r3ld3v Nice catch, thanks! Also, good to see you here. :)

**Comment by str4d** [comment]
Post-hoc utACK.

**Pull request merged**